src/ILCompiler/src/RemovingILProvider.cs

```diff
@@ -143,6 +143,20 @@ private RemoveAction GetAction(MethodDesc method)
                         return RemoveAction.ConvertToThrow;
                     }
 
+                    if (method.Name == "GetCalendars" &&
+                        method.Signature.Length == 3 &&
+                        mdType.Name == "CalendarData" && mdType.Namespace == "System.Globalization")
+                    {
+                        return RemoveAction.ConvertToThrow;
+                    }
+
+                    if ((method.Name == "GetTimeFormats" || method.Name == "GetShortTimeFormats") &&
```

Comment:
It is not obvious why these two are stripped when there is a lot others (e.g. `GetLocaleInfo`/`GetLocaleInfoEx`) that can be stripped too. May be add a comment these are important to strip because of they pull in the delegate marshaling support?